frontend/src/metabase/dashboard/components/DashboardEmbedAction/DashboardEmbedAction.tsx

```diff
@@ -0,0 +1,30 @@
+import { useState } from "react";
+import type { Dashboard } from "metabase-types/api";
+import DashboardSharingEmbeddingModal from "metabase/dashboard/containers/DashboardSharingEmbeddingModal";
+import { EmbedMenu } from "../EmbedMenu";
+
+export const DashboardEmbedAction = ({
+  dashboard,
+}: {
+  dashboard: Dashboard;
+}) => {
+  const [isModalOpen, setIsModalOpen] = useState(false);
+
+  return (
+    <>
+      <EmbedMenu
+        resource={dashboard}
+        resourceType="dashboard"
+        hasPublicLink={!!dashboard.public_uuid}
```

Comment:
Yeah so this will make more sense when we have to add the public popover, since there's big differences between questions and dashboards regarding the redux actions, UUID access (`question.publicUUID()` vs `dashboard.public_uuid`), and getting the public link. It'll make more sense with the public link PR, since they'll take advantage of using the `resource` prop a little more clearly